### PR TITLE
Add code from prototype.

### PR DESCRIPTION
**Pull request by jeffmendoza**

### CODE REVIEW

File: pkg/policies/branch/branch.go [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+	"net/http"
+	"path"
+
+	"github.com/ossf/allstar/pkg/config"
+	"github.com/ossf/allstar/pkg/policydef"
+
+	"github.com/google/go-github/v35/github"
+)
+
+const config_ConfigFile = "branch_protection.yaml"
```
Comment (dlorenc) [inline pkg/policies/branch/branch.go:31]: nit: I think linters will eventually complain about these _'s in names. I think something like `const configFile` is probably fine

File: pkg/issue/issue.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	// check pagination
+	is, _, err := c.Issues.ListByRepo(ctx, owner, repo, opt)
+	if err != nil {
+		return err
+	}
+	var issue *github.Issue
+	t := fmt.Sprintf(title, policy)
+	for _, i := range is {
+		if i.GetTitle() == t {
+			issue = i
```
Comment (dlorenc) [inline pkg/issue/issue.go:48]: should we break here?

File: pkg/issue/issue.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+			PerPage: 100,
+		},
+	}
+	// check pagination
+	is, _, err := c.Issues.ListByRepo(ctx, owner, repo, opt)
+	if err != nil {
+		return err
+	}
+	var issue *github.Issue
+	t := fmt.Sprintf(title, policy)
```
Comment (dlorenc) [inline pkg/issue/issue.go:45]: nit: logging instead of fmt?

File: pkg/issue/issue.go [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+		ListOptions: github.ListOptions{
+			PerPage: 100,
+		},
+	}
+	// check pagination
+	is, _, err := c.Issues.ListByRepo(ctx, owner, repo, opt)
+	if err != nil {
+		return err
+	}
+	var issue *github.Issue
```
Comment (dlorenc) [inline pkg/issue/issue.go:100]: Maybe refactor this to make a helper? I see the issue loop in a few places.

**Review by dlorenc — COMMENTED** [review]
A couple nits, nice work!

File: pkg/policies/branch/branch.go (hunk 1 above)
Comment (jeffmendoza) [reply pkg/policies/branch/branch.go:31]: I put "config_" in front of anything that I want to pull out into a k8s configmap and not leave in as a const

File: pkg/issue/issue.go (hunk 3 above)
Comment (jeffmendoza) [reply pkg/issue/issue.go:45]: It's Sprintf

File: pkg/issue/issue.go (hunk 4 above)
Comment (jeffmendoza) [reply pkg/issue/issue.go:101]: Updated comment below with a TODO

File: pkg/issue/issue.go (hunk 3 above)
Comment (dlorenc) [reply pkg/issue/issue.go:45]: d'oh! thanks.

**Review by dlorenc — APPROVED** [review]

**Pull request merged**